src/styles/main.scss

```diff
@@ -1,3 +1,211 @@
+$main-col: #00acdc;
+$dark-col: #060b35;
+$text-col: #616070;
+$default-col: #fff;
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  color: $text-col;
+}
+
 body {
   margin: 0;
 }
+
+.cards {
+  display: flex;
+  flex-wrap: wrap;
+  box-sizing: border-box;
+  width: 1024px;
+  margin: 0 auto;
+  padding: 50px 40px;
+}
+
+.cards__card {
+  margin-right: 48px;
+  margin-bottom: 46px;
+}
+
+.cards__card:nth-child(4n) {
+  margin-right: 0;
+}
+
+.cards__card:nth-last-child(-n + 4) {
+  margin-bottom: 0;
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  background: $default-col;
```

Comment:
better to use scss nesting

Reply:
https://sass-lang.com/documentation/style-rules/parent-selector/#adding-suffixes